basicswap/interface/firo.py

```diff
@@ -102,6 +102,70 @@ def isAddressMine(self, address: str, or_watch_only: bool = False) -> bool:
             return addr_info["ismine"]
         return addr_info["ismine"] or addr_info["iswatchonly"]
 
+    def getNewSparkAddress(self) -> str:
+        """Generate a new Spark address for receiving private funds.
+        RPC: getnewsparkaddress
+        """
+        try:
+            return self.rpc_wallet("getnewsparkaddress")
```

Comment:
```suggestion
            return self.rpc_wallet("getnewsparkaddress")[0]
```

This endpoint expects a string but returns an array

Reply:
fix those changes